app/views/events/_applicants_overview.html.erb

```diff
@@ -13,7 +13,7 @@
         <%= sort_caret(Profile.human_attribute_name(:gender), :gender) %>
       </th>
       <th>
-        <%= sort_caret(Profile.human_attribute_name(:age), :age) %>
+        <%= sort_caret(( t('.age') + " <br> " + t('.when_event_starts')), :applicant_age_when_event_starts ) %>
```

Comment:
I think the problem is the <br> tag in general. The HTML should describe the structure of the document independent of the visual representation. There are some cases where a line break has semantic meaning like in adresses or poetries. But I think this isn't such a case and therefore the positioning should be handled with CSS files?
I don't know how to archive this with CSS myself, but probably there exists some CSS magic ;) For now I think it isn't that bad that it is on one line if the room is available. Otherwise it should break the line either way?